### PR TITLE
[UnitTest] adding more properties testing

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: engines/python/setup/djl_python/tests/test_properties_manager.py [hunk 1]
```diff
@@ -101,6 +101,38 @@ def test_tnx_configs(self):
...
+            "load_split_model":
+            "true",
+            "load_in_8bit":
+            "true",
+            "compiled_graph_path":
+            "s3://test/bucket/folder",
+            "low_cpu_mem_usage":
+            "true",
+            'context_length_estimate':
+            '{"context_length": "128", "variable_size": "12"}'
```
Comment (sindhuvahinis) [inline engines/python/setup/djl_python/tests/test_properties_manager.py:119]: @tosterberg Added temp context_length_estimate for testing whether we are parsing the properties.

**Review by tosterberg — APPROVED** [review]
